Packages/ChatSecure/ChatSecure/Classes/Signal/InMemoryStore/IdentityKeyInMemoryStore.swift

```diff
@@ -31,10 +31,13 @@ final class IdentityKeyInMemoryStore {
 			let currrentServer = channelStorage.realmManager.getCurrentServer()
 			let clientId = currrentServer?.profile?.userId ?? ""
 			let domain = currrentServer?.serverDomain ?? ""
+			if let tempKey = identityKeys[clientId+domain] {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/0b09249675dff5b0a460b69f27e9f3e1d518728a/Packages/ChatSecure/ChatSecure/Classes/Signal/InMemoryStore/IdentityKeyInMemoryStore.swift#L34"></span>Operators should be surrounded by a single whitespace when they are being used.<br />
<code>operator_usage_whitespace</code> <code>IdentityKeyInMemoryStore.swift:34</code></td>
    </tr>
  </tbody>
</table>